### ISSUE TITLE
Travis build broken

### ISSUE THREAD

**Issue by Louis-Backstrom** [bird-team/brisbane-bird-atlas#137]
Can't work this one out - no obvious error message that I can see in 09b93ee. Here's the end of the log:

````
Quitting from lines 32120-32121 (brisbane-bird-atlas.Rmd) 
Error in if (is.na(path)) path <- "assets/misc/missing-profile.png" : 
  argument is of length zero
Calls: system.time ... withVisible -> eval -> eval -> render_species_profile
In addition: Warning message:
In data.table::fread(record_path, data.table = FALSE) :
  Found and resolved improper quoting out-of-sample. First healed line 31517: <<URN:CornellLabOfOrnithology:EBIRD:OBS680234213	2018-11-26 17:52:00.0	25487	species	Silver-eye	Zosterops lateralis			X				Australia	AU	Queensland	AU-QLD	Brisbane	AU-QLD-BRI					Toohey Forest Conservation Park	L2559772	H	-27.54115	153.05191	1988-04-13	06:10:00	obsr302493	S50190935	Historical	P62	EBIRD				1	0		0	1	0		Birding near our quarters in the QE2 stadium in trails around the Toohey Forest and the Griffith campus	"Seen in small groups in the woods" and seen regularly around QE II for the rema>>. If the fields are not quoted (e.g. field separator does not appear within any field), try quote="" to avoid this warning.

Timing stopped at: 61.68 4.188 91.61
Execution halted
bba
bba
ls: cannot access _book/brisbane-bird-atlas.pdf: No such file or directory
make: *** [book_pdf] Error 2
/home/travis/.travis/functions: line 524:  4244 Terminated              travis_jigger "${!}" "${timeout}" "${cmd[@]}"
travis_time:end:0fa76c2d:start=1554633214142144912,finish=1554633311694566435,duration=97552421523
[0K[31;1mThe command "travis_wait 120 make book_pdf" exited with 2.[0m

travis_time:start:10f3218f
[0K$ if [ "${TRAVIS_BRANCH}" != "master" ]; then travis_terminate 0; fi
travis_time:end:10f3218f:start=1554633311725211531,finish=1554633311736122840,duration=10911309
[0K[32;1mThe command "if [ "${TRAVIS_BRANCH}" != "master" ]; then travis_terminate 0; fi" exited with 0.[0m

travis_time:start:0e519ce5
[0K$ make deploy_book_pdf
6fb27205110a9f84726e02e4a70518f41c1418912946702f88a47b8caf0b3c88
Warning message:
In pb_upload_file(f, repo, tag, name, overwrite, use_timestamps,  :
  file _book/brisbane-bird-atlas.pdf does not exist
bba
bba
ls: cannot access _book/brisbane-bird-atlas.pdf: No such file or directory
make: *** [deploy_book_pdf] Error 2
travis_time:end:0e519ce5:start=1554633311742438876,finish=1554633316741752229,duration=4999313353
[0K[31;1mThe command "make deploy_book_pdf" exited with 2.[0m



Done. Your build exited with 1.
````

Any ideas @jeffreyhanson? Nothing I can think of in my recent commits (last successful book build was cb10ac6) should be throwing a spanner in the works.

E: The only thing I can think of is that some of the new species might be causing issues - I'm not sure that they all currently have data associated with them in the eBird data (e.g. metallic starling / aplonis metallica and similar others we've only recently put into eBird). This didn't give me too many hiccups for the asset building (except for those species which I've "stashed") but I guess maybe it is for the book?

eBird March update is due soon, which should have most if not all of the new species so hopefully if that is the problem is readily fixed in ten days or so.

**Comment by Louis-Backstrom** [comment]
Out of my depth still - just pushed a bunch of commits that were intended to fix this issue but the end result (as of adfce1a) is still the same. Am I forgetting something?

**Comment by jeffreyhanson** [comment]
Yeah, I'm not sure either. I think it's caused by a species that has an rmarkdown page but is missing from the `atlas_list.xlsx` file (or maybe it's mispelled in the `atlas_list.xlsx` file or the rmarkdown page). I've just pushed a change which will print the species name in the Travis build log, so that we can work out how to fix this.

**Comment by jeffreyhanson** [comment]
The "atlas_list.xlsx" contains a record for the species "Cacatua tenuirostris x sanguinea" and the "Cacatua-tenuirostris-x-sanguinea.Rmd" file contains the name "Cacatua sanguinea x tenuirostris". @Louis-Backstrom or @dbl3raf, which name should be used to refer to this group of animals (for lack of a better term)? It will need to be consistent across all files. I have changed it to "Cacatua tenuirostris x sanguinea" for consistency, but please feel free to change everything to  "Cacatua sanguinea x tenuirostris" if needed.

**Comment by jeffreyhanson** [comment]
This fixed the Travis issues so the pdf book is now working again. I'll leave this open in case @Louis-Backstrom or @dbl3raf need to change the name.

**Comment by Louis-Backstrom** [comment]
That taxon has been the death of me...

It should be Cacatua tenuirostris x sanguinea (as per eBird taxonomy) but for some reason it was originally in the other way round - not sure if that's a taxonomy change or an error when we first put it onto the list or what. I thought I had fixed all the instances up but apparently not. Given that the build is working now I'm happy to close this.

I'll unstash the species I recently stashed once we update to the March dataset (probably 10 days or so).
